### PR TITLE
Added a basic Back to top of page button

### PR DESCRIPTION
**Pull request by garvitkhandelwal**
Changes have to be made to this button to make it look nice but it can be used on a temporary basis.

### CODE REVIEW

File: css/custom.css [hunk 1]
```diff
@@ -134,4 +134,24 @@ footer a:hover{
 
 .icon{
   padding-right: 5px;
-}
\ No newline at end of file
+}
+
+#myBtn {
```
Comment (CloudyPadmal) [inline css/custom.css]: It would be better if you can name it like `backButton` instead of `myBtn` :)

File: index.html [hunk 2]
```diff
@@ -461,6 +461,13 @@ <h3 id="">Georges Khazandar</h3>
                         </div>
                     </div>
                     <hr>
+
+                    <!--- Back to top button --->
```
Comment (CloudyPadmal) [inline index.html]: Comments use only 2 dashes.

Comment (garvitkhandelwal) [reply index.html]: Changes done. Thanks for telling.

File: css/custom.css (hunk 1 above)
Comment (garvitkhandelwal) [reply css/custom.css]: I renamed it back-to-top button

File: css/custom.css [hunk 3]
```diff
@@ -134,4 +134,24 @@ footer a:hover{
...
+    color: black; /* Text color */
+    cursor: pointer; /* Add a mouse pointer on hover */
+    padding: 15px; /* Some padding */
+    border-radius: 10px; /* Rounded corners */
+}
+
+#myBtn:hover {
+    background-color: #FFFFFF; /* Add a dark-grey background on hover */
+}
+
```
Comment (aashutoshrathi) [inline css/custom.css:157]: Add newline at end of file

**Review by aashutoshrathi — CHANGES_REQUESTED** [review]
Provide a deployment link and screenshots, so that one can review changes

**Comment by aashutoshrathi** [comment]
Ping @garvitkhandelwal are you on it?

**Pull request closed without merging**